tests/OpenQasmExporter.Tests/OpenQasmExporter.Tests.csproj

```diff
@@ -0,0 +1,52 @@
+﻿<Project Sdk="Microsoft.Quantum.Sdk/0.12.20070124">
+
+  <PropertyGroup>
+	<OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+	  <DefaultSimulator>Microsoft.Quantum.Samples.OpenQasmExporter.OpenQasmExporter</DefaultSimulator>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\OpenQasmExporter\OpenQasmExporter.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="JetBrains.DotMemoryUnit" Version="3.1.20200127.214830" />
```

Comment:
That library is a small issue in open source projects. You are not using it in your code, but since you have resharper installed its probably not visible.